pkg/providers/vsphere/vsphere.go

```diff
@@ -149,22 +142,34 @@ type ClusterResourceSetManager interface {
 	ForceUpdate(ctx context.Context, name, namespace string, managementCluster, workloadCluster *types.Cluster) error
 }
 
+type ValidatorBuilder interface {
```

Comment:
Is it possible to avoid a `ValidatorBuilder` now that #2887 is merged?

Reply:
Oh i meant to strip this out, yes 👍